engine/installation/linux/suse.md

```diff
@@ -149,13 +180,25 @@ the repository.
     $ sudo zypper install docker-ee-<VERSION_STRING>
     ```
 
-    Start Docker:
+4.  Configure Docker to use the `btrfs` filesystem. Edit the file
```

Comment:
They don't, really. But they would if their `/var/lib/docker/` wasn't BTRFS or if `/` was not BTRFS, so it made sense to ask them to do it. Customers might ship the `daemon.json` around too.